build.sbt

```diff
@@ -4,7 +4,6 @@ ThisBuild / tlBaseVersion := "0.6"
 ThisBuild / developers += tlGitHubDev("erikerlandson", "Erik Erlandson")
 ThisBuild / organization := "com.manyangled"
 ThisBuild / organizationName := "Erik Erlandson"
-ThisBuild / licenses := List("Apache-2.0" -> url("http://opensource.org/licenses/Apache-2.0"))
```

Comment:
Keeping this is harmless of course, but I omitted it because it's already the default :)

Reply:
Oh, unless you prefer that specific URL. sbt-typelevel uses `https://www.apache.org/licenses/LICENSE-2.0.txt`

Reply:
I'd like to keep the license setting, however `https://www.apache.org/licenses/LICENSE-2.0.txt` seems like a better url since it is from apache.org

Reply:
Done!